Parallel does not build

I was following the instructions, and the build stopped at "building parallel". 

```
$ cat logs/parallel_2_build.log 
make  all-recursive
make[1]: Entering directory `/home/miquel/opt/OpenDroneMap/src/parallel'
Making all in src
make[2]: Entering directory `/home/miquel/opt/OpenDroneMap/src/parallel/src'
pod2man --release='20100922' --center='parallel' \
        --section=1 ./parallel > ./parallel.1
pod2man --release='20100922' --center='parallel' \
        --section=1 ./sem.pod > ./sem.1
pod2man --release='20100922' --center='parallel' \
        --section=1 ./sql > ./sql.1
pod2html ./parallel > ./parallel.html
pod2html ./sem.pod > ./sem.html
pod2html ./sql > ./sql.html
rm -f ./pod2htm*
rm -f ./pod2htm*
./parallel around line 334: Expected text after =item, not a number
./parallel around line 340: Expected text after =item, not a number
rm -f ./pod2htm*
./parallel around line 1510: Expected text after =item, not a number
./parallel around line 1514: Expected text after =item, not a number
POD document had syntax errors at /usr/bin/pod2man line 71.
make[2]: *** [parallel.1] Error 255
make[2]: Leaving directory `/home/miquel/opt/OpenDroneMap/src/parallel/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/miquel/opt/OpenDroneMap/src/parallel'
make: *** [all] Error 2
```

As suggested in the fix for a [similar problem](http://comments.gmane.org/gmane.network.inn/9841), the solution is to put "Z<>" in the "=item" argument string.

--- a/install.sh
+++ b/install.sh
@@ -121,7 +121,7 @@ do
     echo "    - already downloaded $source"
   fi
 done <<EOF
-parallel.tar.bz2  http://ftp.gnu.org/gnu/parallel/parallel-20100922.tar.bz2
+parallel.tar.bz2  http://ftp.gnu.org/gnu/parallel/parallel-20141022.tar.bz2
 clapack.tgz  http://www.netlib.org/clapack/clapack-3.2.1-CMAKE.tgz
 bundler.zip  http://phototour.cs.washington.edu/bundler/distr/bundler-v0.4-source.zip
 PoissonRecon.zip http://www.cs.jhu.edu/~misha/Code/PoissonRecon/Version2/PoissonRecon.zip
@@ -152,7 +152,7 @@ mv -f graclus1.2			    "$GRACLUS_PATH"
 mv -f clapack-3.2.1-CMAKE "$CLAPACK_PATH"
 mv -f vlfeat-0.9.13       "$VLFEAT_PATH"
 mv -f bundler-v0.4-source "$BUNDLER_PATH"
-mv -f parallel-20100922   "$PARALLEL_PATH"
+mv -f parallel-20141022   "$PARALLEL_PATH"
 mv -f PoissonRecon        "$PSR_PATH"
 mv -f cmvs                "$CMVS_PATH"
 

